mix.exs

```diff
@@ -30,7 +30,8 @@ defmodule Scidata.MixProject do
 
   defp deps do
     [
-      {:ex_doc, ">= 0.24.0", only: :dev, runtime: false}
+      {:ex_doc, ">= 0.24.0", only: :dev, runtime: false},
+      {:csv, "~> 2.4"}
```

Comment:
Can you please use `nimble_csv` for CSV parsing? :)

Reply:
Thanks for the suggestion! I've made the required change.